test/release/examples/primers/FFTWlib.compopts

```diff
@@ -1 +1,9 @@
--I$FFTW_DIR/include -L$FFTW_DIR/lib -sautoInitFFTW_MT=false  -lfftw3
+#!/usr/bin/env bash
+system=$($CHPL_HOME/util/printchplenv | grep 'CHPL_TARGET_PLATFORM' | cut -d\  -f2)
```

Comment:
Not worth changing, but FYI that this can just be `system=$CHPL_TARGET_PLATFORM` (the test infra will stuff the printchplenv environment into this subprocess)

Or if you want it to work outside the testing infra it could be `system=$($CHPL_HOME/util/chplenv/chpl_platform.py --target)`

Reply:
Good to know, I thought that it should be possible, but I found an example that does this in the existing tests first, so I stuck with it.